metric-anomaly-detector/src/main/java/org/hypertrace/alert/engine/metric/anomaly/detector/evaluator/AlertRuleEvaluator.java

```diff
@@ -78,8 +89,22 @@ public AlertRuleEvaluator(Config appConfig) {
 
     switch (violationCondition.getConditionCase()) {
       case STATIC_THRESHOLD_CONDITION:
+        staticRuleCounter
```

Comment:
I think below two metrics are not giving much information. As most of the time, it will be static. 
- count of static rules evaluated
- count of dynamic rules evaluated
I if we want, we can combine this with our latency metrics by adding a tag (static / baseline). So, we don't have to create new metrics, but still be able to do check the same metrics.